omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -2775,13 +2813,13 @@
                                                 <div class="row">
                                                     <div class="col-md-4">
                                                         <br/>
-                                                        <obs conceptId="1282" answerConceptSetIds="160021"
+                                                        <obs id="b6-trigger" conceptId="1282" answerConceptSetIds="160021"
                                                              style="radio"
                                                              answerConceptIds="82912" answerLabels="Vitamin B6"/>
                                                     </div>
                                                     <div class="col-md-4">
                                                         <label>No. of Pills</label>
-                                                        <obs conceptId="160856"/>
+                                                        <obs id="b6-pills" conceptId="160856"/>
```

Comment:
@dbaluku rename this field to vitamin-b6 (for clarity - had confused me earlier)

Reply:
Done